specs/lib/removeUnusedSkinsSpec.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+var removeUnusedSkins = require('../../').removeUnusedSkins;
+var OptimizationStatistics = require('../../').OptimizationStatistics;
+
+describe('removeUnusedSkins', function() {
+    it('removes a skin', function() {
+        var gltf = {
+            "nodes": {
+                "Cylinder": {
+                    "children": [],
+                    "matrix": [
+                        1,
+                        0,
+                        0,
+                        0,
+                        0,
+                        1,
+                        0,
+                        0,
+                        0,
+                        0,
+                        1,
+                        0,
+                        0,
+                        0,
+                        0,
+                        1
+                    ],
+                    "meshes": [
+                        "Cylinder-mesh"
+                    ],
+                    "name": "Cylinder",
+                    "skeletons": [
+                        "Bone"
+                    ],
+                    "skin": "Armature_Cylinder-skin"
+                }
+            },
+            "skins": {
+                "Armature_Cylinder-skin": {
+                    "bindShapeMatrix": [
```

Comment:
Remove the `bindShapeMatrix`, it defaults to identity.  https://github.com/KhronosGroup/glTF/blob/master/specification/README.md#skinbindshapematrix